### PR TITLE
Add Open eCard App - Open Source implementation of ISO/IEC 24727

### PR DESCRIPTION
**Pull request by jatty**
Please include my new cask Open eCard App
Open Source implementation of ISO/IEC 24727authentication and signature purposes for SmartCards

This is my first contribution, please be kind :-)

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
  1 offense: "https://www.openecard.org/download/pc/ does not match github.com/ecsec/open-ecard/releases/download/„
  this is because the project homepage download page directs to image which is located at GitHub
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version]

new cask:

- [ ] Named the cask according to the [token reference].
  not sure?!
- [x] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
   hopefully zap is complete (or should we also remove ~/.openecard/ completely?)
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

**Review by jatty — COMMENTED** [review]
Please ignore this pull request. this was erroneous. See my next one.

File: Casks/open-ecard.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  # github.com/ecsec/open-ecard was verified as official when first introduced to the cask
+  url "https://github.com/ecsec/open-ecard/releases/download/#{version}/Open-eCard-App-#{version}.dmg"
+  appcast 'https://github.com/ecsec/open-ecard/releases.atom'
+  name 'Open eCard'
+  homepage 'https://www.openecard.org/'
+
+  pkg 'Open eCard App.pkg'
+
+  uninstall delete: "#{appdir}/Open-eCard-App.app"
```
Comment (vitorgalvao) [inline Casks/open-ecard.rb]: Is this really all the app does? Then why a `pkg`?

Comment (jatty) [reply Casks/open-ecard.rb]: I'm not the author of the package, so I'm not sure, but with a "find . -mtime 1" I don't get more directories other than the two ones for "uninstall" and "zap"

Comment (vitorgalvao) [reply Casks/open-ecard.rb]: `pkg`s can do anything; a `find` doesn’t suffice. See [the documentation](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md).

In this case:

> Error: Some packages are still installed, add them to uninstall pkgutil:
com.openecard.pkg.OpeneCardApp

Comment (jatty) [reply Casks/open-ecard.rb]: Thanks, now i get the clue. I'll followup with another pull request very soon.

Comment (vitorgalvao) [reply Casks/open-ecard.rb]: > I'll followup with another pull request very soon.

Don’t. Edit this one.

**Comment by jatty** [comment]
@vitorgalvao Thanks, I'm glad we got it to this point. :-)

**Comment by vitorgalvao** [comment]
Looks good. Thank you, and welcome.

**Pull request merged**